storage/aptosdb/src/pruner/ledger_pruner/mod.rs

```diff
@@ -67,10 +69,13 @@ impl DBPruner for LedgerPruner {
             self.ledger_metadata_pruner
                 .prune(progress, current_batch_target_version)?;
 
-            // NOTE: If necessary, this can be done in parallel.
-            self.sub_pruners
-                .iter()
-                .try_for_each(|pruner| pruner.prune(progress, current_batch_target_version))?;
+            THREAD_MANAGER.get_io_pool().install(|| {
```

Comment:
wouldn't these interfere with high pri io gets?

Reply:
This is cheaper than the io_pool operations in commit phase, and the IO pool has 64 threads in the default strategy, so I don't worrying about it for now. Unless you have strong opinion to have another low pri thread pool, and set the priority on threads right now.

Reply:
Offline discussed, added a background pool.